_security/access-control/api.md

```diff
@@ -36,6 +36,22 @@ plugins.security.restapi.endpoints_disabled.<role>.<endpoint>: ["<method>", ...]
 ```
 {% include copy.html %}
 
+Roles also allow you to control access to specific REST APIs. You can add individual or multiple cluster permissions to a role and grant users access to associated APIs when they are mapped to the role. The following list of cluster permissions includes the endpoints that correspond to the Security REST APIs:
+
+| **Permission**                 | **APIs Granted**                 | **Description**                                                                                                                                   |
+|:-------------------------------|:---------------------------------|:--------------------------------------------------------------------------------------------------------------------------------------------------|
```

Comment:
You don't need to add extra spaces or "filler" dashes to make tables render properly in the markdown. And the headings should automatically be boldface, too, without having to add the double asterisks. This should do it:
\| Permission | APIs Granted | Description |
\| :--- | :--- | :--- |
\| Permission 1 | `allowlist` | Does this and that |

| Permission | APIs Granted | Description |
| :--- | :--- | :--- |
| Permission 1 | `allowlist` | Does this and that |

Reply:
These were auto-generated by my IDE. I guess for better readability